lib/Customer.php

```diff
@@ -202,4 +203,54 @@ public static function allSources($id, $params = null, $opts = null)
     {
         return self::_allNestedResources($id, static::PATH_SOURCES, $params, $opts);
     }
+
+    /**
+     * @param string|null $id The ID of the customer on which to create the tax id.
+     * @param array|null $params
+     * @param array|string|null $opts
+     *
+     * @return ApiResource
+     */
+    public static function createTaxId($id, $params = null, $opts = null)
```

Comment:
Flagging that `tax_ids` the collection is returned on the Customer object, like sources and subscriptions. Not sure anymore if that changes things